log/zerolog.go

```diff
@@ -0,0 +1,73 @@
+package log
+
+import (
+	"os"
+	"time"
+
+	cmtlog "github.com/cometbft/cometbft/libs/log"
+	"github.com/rs/zerolog"
+)
+
+// Defines commons keys for logging
+const ModuleKey = "module"
+
+var (
+	// ContextKey is used to store the logger in the context
+	ContextKey struct{}
+	_          Logger = (*ZeroLogWrapper)(nil)
+)
+
+func NewZeroLogger(key, value string) *zerolog.Logger {
+	output := zerolog.ConsoleWriter{Out: os.Stdout, TimeFormat: time.Kitchen}
+	logger := zerolog.New(output).With().Str(key, value).Timestamp().Logger()
+	return &logger
+}
+
+// TODO: add filtered logging in ZeroLog: https://github.com/cosmos/cosmos-sdk/pull/13236 / https://github.com/cosmos/cosmos-sdk/issues/13699#issuecomment-1354887644
+
+// ZeroLogWrapper provides a wrapper around a zerolog.Logger instance. It implements
+// Tendermint's Logger interface.
+type ZeroLogWrapper struct {
```

Comment:
Why do we have a `ZeroLogWrapper`? This defeats the purpose of using zerolog (with the way it's currently implemented).

Reply:
This actually re-adds it as it was in server before this PR: https://github.com/cosmos/cosmos-sdk/pull/13236.
So then we can re-add filtering. I have not changed the implementation.

Reply:
Yes yes, I recall (I wrote it :P) but what I'm asking is that we removed it for a reason and now we're adding it back. Why are we adding it back? It's not performant nor does it give us any of the benefits of zerolog

Reply:
I thought it was removed because it didn't have filtered logging.
It was still more performant that the TM logger and re-added it to have a base.
If it should be deleted, then sure!

Reply:
Yes, the filtering was missing indeed. But I'm pretty sure it was less performant I think. There's an issue lingering around

Reply:
we will add filter logging before creating v1 of the log package.

Reply:
I'd like to see 1 basic logger (what Comet provides and we mimicked) and another, more powerful logger interface that supports typed and filtered logging :)

Reply:
why not just have one logger in general. The interface provided by comet will be replaced to what we want after we migrate the repo. its only the same interface cause it would be a unreviewable pr so this way its piece meal

Reply:
> The interface provided by comet will be replaced to what we want after we migrate the repo.

How? You've convinced them to adopt our logging interface?

Reply:
>  You've convinced them to adopt our logging interface?

I was thinking we do a wrapper of our logger where we pass it to tendermint. there is no need for us to have the same interface. 

https://github.com/cosmos/cosmos-sdk/issues/14967

Reply:
Yes, but at some point you will have to pass a logger to Comet and that logger must conform to Comet's interface. In essence, we'll have two loggers being used -- one for Comet and one for the app.

Reply:
yes, at that point we just provide a wrapper that is separate from our interface.

Reply:
cc @julienrbrt we should call the logger `CometZerologWrapper` IMO